infra/main.tf

```diff
@@ -2,6 +2,7 @@ locals {
   tags            = { ProjectName = "Information Assistant", BuildNumber = var.buildNumber }
   azure_roles     = jsondecode(file("${path.module}/azure_roles.json"))
   selected_roles  = ["CognitiveServicesOpenAIUser", "StorageBlobDataReader", "StorageBlobDataContributor", "SearchIndexDataReader", "SearchIndexDataContributor"]
+  backend_name    = var.backendServiceName != "" ? var.backendServiceName : "infoasst-web-${random_string.random.result}"
```

Comment:
This shouldn't be needed anymore with the removal of SL

Reply:
removed as well.